src/emailUnpackFiles/sendEmail.js

```diff
@@ -0,0 +1,45 @@
+// function to send email peer to peer
+
+import axios from 'axios';
+
+// import the keys from .env?
```

Comment:
Perhaps you should set these keys in .env then import them into this file, that way we can just update .env as needed instead of refactoring this code later.

If you add them to .env, please also add the key, but not the value, to the Sample.env file